build/xtask/src/dist.rs

```diff
@@ -2138,6 +2156,95 @@ fn write_srec(
     Ok(())
 }
 
+fn write_elf(
```

Comment:
We will probably need write_elf32 and write_elf64 if we have to support consistent flows for all architectures.

Reply:
I feel like it should be able to share most of the code.  From what I have seen, alot of the `goblin::elf` apis can be agnostic to if it is 32 or 64 bit.

Reply:
Agree w/ @beezow, I suspect this will be relatively straightforward.

Reply:
Changed in latest revision.